include/linux/vhm/acrn_common.h

```diff
@@ -496,6 +496,26 @@ struct acrn_pcidev {
 
 } __aligned(8);
 
+/**
+ * @brief Info to assign or deassign a MMIO device for a VM
+ *
+ * the parameter for HC_ASSIGN_MMIODEV or HC_DEASSIGN_MMIODEV hypercall
+ */
+struct acrn_mmiodev {
+	/** the gpa of the MMIO region for the MMIO device */
+	uint64_t base_gpa;
+
+	/** the hpa of the MMIO region for the MMIO device */
+	uint64_t base_hpa;
+
+	/** the size of the MMIO region for the MMIO device */
+	uint64_t size;
+
+	/** reserved for extension */
+	uint64_t reserved[13];
```

Comment:
any rule of reserved room for the structure? why 13?

Reply:
This structure is 8 bytes aligned.

Reply:
> This structure is 8 bytes aligned.

uint64_t is already 8 bytes aligned.

Reply:
Sorry, I hope the structure to be 2 ^ n aligned not only 8 bytes aligned.

Reply:
Besides, we may need to support more mmio regions and some fields needs to be reserved for SID/HID (meybe)/ IRQ.
So I hope we could reserve as much memory as possible.

Reply:
OK.

Reply:
Can you add a comment about this for the API documentation.

Reply:
@dbkinder  You mean add  a comment for the reserved  field ?

Reply:
That would be an ok place to mention the alignment expectations (and help explain why the value is currently 13)